Cargo.toml

```diff
@@ -14,11 +14,13 @@ edition = "2018"
 bytes = "~0.4.12"
 config_file_handler = "~0.11.0"
 crossbeam-channel = "~0.3.9"
+ctrlc = { version = "~3", optional = true, features = ["termination"] }
```

Comment:
Is that not a dev dependency?

Reply:
No, because the code that depends on this lives in `src` and is not exclusively behind `cfg(test)`.

Reply:
I do have an idea how to make it possible though. Instead of adding this seed printing functionality to `mock::quic_p2p::Network`, we would create a new type, say `TestEnv` which would live in `tests/mock_network_utils.rs` and which would contain the mock-quic-p2p `Network` as well as the seed printing stuff. Then we would use this `TestEnv` everywhere we currently use `Network` in the mock network tests. Might be better to do it in a followup PR (if at all), to keep this one small and focused (as much as possible).